gene2phenotype_project/gene2phenotype_app/utils/curationinfo_utils.py

```diff
@@ -61,8 +61,11 @@ def subject_confidence(self)-> str:
 
         Returns:
             str: Subject string
-        """        
-        return f"Updated confidence for {self.stable_id}"
+        """
+        subject_string = f"Updated confidence for {self.stable_id}"
+        if self.host != "ebi.ac.uk":
```

Comment:
With this check any update on wwwdev.ebi.ac.uk is not going to include the subject `[THIS IS A TEST]`.
We don't want an exact match, we want to check if string `www.ebi.ac.uk` is not found in `self.host`.

Reply:
Actually it should include the this is a test for any update on wwwdev.ebi.ac.uk, i have updated the check to actually check on www.ebi.ac.uk

Reply:
When I test locally the host is `http://localhost:8000`
So `if self.host != "wwwebi.ac.uk"` probably is not going to work.